easybuild/easyconfigs/p/Python/Python-3.8.4-GCCcore-10.2.0.eb

```
@@ -0,0 +1,428 @@
+    }),
+    ('filelock', '3.0.12', {
+        'checksums': ['18d82244ee114f543149c66a6e0c14e9c4f8a1044b5cdaadd0f82159d6a6ff59'],
+    }),
+    ('virtualenv', '20.0.34', {
+        'checksums': ['4bf0e2bf99d33b123a895a5a244f0d7adb2a92171c6bbb31c3e2db235624abf1'],
+    }),
+    ('docopt', '0.6.2', {
+        'checksums': ['49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491'],
+    }),
+    ('joblib', '0.17.0', {
+        'checksums': ['9e284edd6be6b71883a63c9b7f124738a3c16195513ad940eae7e3438de885d5'],
+    }),
+    ('chardet', '3.0.4', {
+        'checksums': ['84ab92ed1c4d4f16916e05906b6b75a6c0fb5db821cc65e70cbd64a3e2a5eaae'],
+    }),
+    ('certifi', '2020.6.20', {
+        'checksums': ['5930595817496dd21bb8dc35dad090f1c2cd0adfaf21204bf6732ca5d8ee34d3'],
+    }),
+    ('urllib3', '1.25.10', {
+        'checksums': ['91056c15fa70756691db97756772bb1eb9678fa585d9184f24534b100dc60f4a'],
+    }),
+    ('requests', '2.24.0', {
+        'checksums': ['b3559a131db72c33ee969480840fff4bb6dd111de7dd27c8ee1f820f4f00231b'],
+    }),
+    ('xlrd', '1.2.0', {
+        'checksums': ['546eb36cee8db40c3eaa46c351e67ffee6eeb5fa2650b71bc4c758a29a1b29b2'],
+    }),
+    ('py_expression_eval', '0.3.10', {
+        'checksums': ['43038326b686df697f9533895184c15d18769e215abbd8bcecaea607483f35b3'],
+    }),
+    ('tabulate', '0.8.7', {
+        'checksums': ['db2723a20d04bcda8522165c73eea7c300eda74e0ce852d9022e0159d7895007'],
+    }),
+    ('ujson', '4.0.1', {
+        'checksums': ['26cf6241b36ff5ce4539ae687b6b02673109c5e3efc96148806a7873eaa229d3'],
+    }),
+    ('atomicwrites', '1.4.0', {
+        'checksums': ['ae70396ad1a434f9c7046fd2dd196fc04b12f9e91ffb859164193be8b6168a7a'],
+    }),
+    ('py', '1.9.0', {
+        'checksums': ['9ca6883ce56b4e8da7e79ac18787889fa5206c79dcc67fb065376cd2fe03f342'],
+    }),
+    ('scandir', '1.10.0', {
+        'checksums': ['4d4631f6062e658e9007ab3149a9b914f3548cb38bfb021c64f39a025ce578ae'],
+    }),
+    ('pathlib2', '2.3.5', {
+        'checksums': ['6cd9a47b597b37cc57de1c05e56fb1a1c9cc9fab04fe78c29acd090418529868'],
+    }),
+    ('zipp', '3.3.0', {
+        'checksums': ['64ad89efee774d1897a58607895d80789c59778ea02185dd846ac38394a8642b'],
+    }),
+    ('importlib_metadata', '2.0.0', {
+        'checksums': ['77a540690e24b0305878c37ffd421785a6f7e53c8b5720d211b211de8d0e95da'],
+    }),
+    ('pluggy', '0.13.1', {
+        'checksums': ['15b2acde666561e1298d71b523007ed7364de07029219b604cf808bfa1c765b0'],
+    }),
+    ('more-itertools', '8.5.0', {
+        'checksums': ['6f83822ae94818eae2612063a5101a7311e68ae8002005b5e05f03fd74a86a20'],
+    }),
+    ('attrs', '20.2.0', {
+        'modulename': 'attr',
+        'checksums': ['26b54ddbbb9ee1d34d5d3668dd37d6cf74990ab23c828c2888dccdceee395594'],
+    }),
+    ('wcwidth', '0.2.5', {
+        'checksums': ['c4d647b99872929fdb7bdcaa4fbe7f01413ed3d98077df798530e5b04f116c83'],
+    }),
+    ('pytest', '6.1.1', {
```

`pytest` is missing a dependency on `iniconfig`
```suggestion
    ('iniconfig', '1.1.1', {
        'checksums': ['bc3af051d7d14b2ee5ef9969666def0cd1a000e121eaea580d4a313df4b37f32'],
    }),
    ('pytest', '6.1.1', {
```